package/yaft/package

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(yaft)
+pkgdesc="Yet another framebuffer terminal"
+url=https://github.com/timower/rM2-stuff/tree/master/apps/yaft
+pkgver=0.0.4-1
+timestamp=2021-04-11T10:42Z
+maintainer="None <none@example.com>"
+license=GPL-3.0
+section="admin"
+image=base:v1.6
+
+source=(https://github.com/timower/rM2-stuff/archive/refs/tags/v0.0.4.tar.gz)
+sha256sums=(dee471ac19ea43ba741f826c9a0a17d7a01bda6472043d400fbcab6fad1931fe)
+
+build() {
+  mkdir build
+  cd build
+  cmake -DCMAKE_TOOLCHAIN_FILE="/usr/share/cmake/$CHOST.cmake" \
+        -DCMAKE_INSTALL_PREFIX="$pkgdir"
```

Comment:
@matteodelabre It seems $pkgdir is not available during build, but I think it should be